bidwire/alembic/versions/c4a6bb537e49_add_description_field_to_document.py

```diff
@@ -0,0 +1,26 @@
+"""Add description field to Document
+
+Revision ID: c4a6bb537e49
+Revises: fbd6ae18c51c
+Create Date: 2017-06-21 19:02:09.414328
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'c4a6bb537e49'
+down_revision = '9b30b0fe231a'
```

Comment:
This is pointing at the wrong revision number, causing the build failure:
```
ERROR [alembic.util.messaging] Multiple head revisions are present for given argument 'head'; please specify a specific target revision, '<branchname>@head' to narrow to a specific head, or 'heads' for all heads
FAILED: Multiple head revisions are present for given argument 'head'; please specify a specific target revision, '<branchname>@head' to narrow to a specific head, or 'heads' for all heads
```

You need to set this to the most recent revision, which I believe is `a1b42c9006a7`.